### ISSUE TITLE
bro week 7

### ISSUE THREAD

**Issue by martiansideofthemoon** [tejas1995/CS213-AlgoAssignments#1]
Week 7 try karna..infi fun, easy and lots to learn :smile:

**Comment by tejas1995** [comment]
Just looked at the assignment, seems very interesting ... Havent done anything post Week3, finally got back to algorithms today after a while. Will make my way through the assignments now, thanks for the heads up! :D

**Comment by martiansideofthemoon** [comment]
yeah anytime :smile: I skipped week 6. Spent 2-3 hours trying to understand it. When I finally did on asking around, give up ho gaya..
